_sections/30-bindings.md

```diff
@@ -112,8 +112,8 @@ A subset of [XPath 1.0 functions](http://www.w3.org/TR/xpath/#corelib), some fun
 <a id="fn:boolean-from-string" href="#fn:boolean-from-string">`boolean-from-string(string arg)`</a> | Returns true if arg is "true" or "1", otherwise returns false.
 <a id="fn:not" href="#fn:not">`not(boolean arg)`</a> | As in [XPath 1.0](http://www.w3.org/TR/xpath/#function-not).
 <a id="fn:number" href="#fn:number">`number(* arg)`</a> | As in [XPath 1.0](http://www.w3.org/TR/xpath/#function-number).
-<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | Converts dateTime value to a number.
-<a id="fn:decimal-time" href="#fn:decimal-time">`decimal-time(time value)`</a> | Converts time value to a number.
+<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | Converts dateTime value to a number in the format used by Excel. The number before the decimal marker is the number of days since January 1, 1970. The number after the decimal marker represents the time as a fractional date.
+<a id="fn:decimal-time" href="#fn:decimal-time">`decimal-time(time value)`</a> | Converts time value to a number representing the number of days since January 1, 1970.
```

Comment:
I do not understand how this is possible... If the parameter was a dateTime value it would make sense, but then it would be the same as date-time.

Reply:
Ohhhhhh. Wow. I'm sorry, for some reason I convinced myself this was `decimal-date`. 😖

Reply:
I'll need to double check to make sure I actually understand the behavior and then fix this. Will probably not be today.